tests/test_identity_map_client.py

```diff
@@ -166,6 +171,20 @@ def assert_unmapped(self, response, reason, dii):
         mapped_identity = response.mapped_identities.get(dii)
         self.assertIsNone(mapped_identity)
 
+    def test_identity_buckets(self):
+        response = self.identity_map_client.get_identity_buckets(datetime.datetime.now() - datetime.timedelta(days=2))
+        self.assertTrue(len(response.buckets) > 0)
+        self.assertTrue(response.is_success)
+
+    def test_identity_buckets_empty_response(self):
+        response = self.identity_map_client.get_identity_buckets(datetime.datetime.now() + datetime.timedelta(days=1))
+        self.assertTrue(len(response.buckets) == 0)
+        self.assertTrue(response.is_success)
```

Comment:
Not sure what information we gain if these tests fail. They could fail because the operator hasn't rotated buckets for a couple of days. Or the clocks are wildly out of sync. Are they giving us more confidence in our code than a unit test would?

Reply:
There's not much we can do. Perhaps I could change it to datetime.datetime.now() - 90 days in test_identity_buckets?